stackslib/src/net/api/getattachmentsinv.rs

```diff
@@ -96,11 +96,13 @@ impl HttpRequest for RPCGetAttachmentsInvRequestHandler {
             if key == "index_block_hash" {
                 index_block_hash = StacksBlockId::from_hex(&value).ok();
             } else if key == "pages_indexes" {
-                if let Ok(pages_indexes_value) = value.parse::<String>() {
-                    for entry in pages_indexes_value.split(',') {
-                        if let Ok(page_index) = entry.parse::<u32>() {
-                            page_indexes.insert(page_index);
-                        }
+                #[allow(clippy::expect_used)]
```

Comment:
Is there a way to remove even this?

Reply:
This is not needed but was there in case in the future we enable clippy to check for uses of `unwrap` and `expect`.

Reply:
Removed with Aaron's fix in e234d37eab1d0c7b10f0430cea9f70a604d5de07